docs/installation.md

```diff
@@ -1,32 +1,64 @@
 # Zonemaster Backend installation guide
 
-This document begins with a number of sections concerning native installation:
+## Overview
 
-* [Prerequisites](#prerequisites)
-* [Dependencies](#dependencies)
-* [Installation](#installation)
-* [Configuration](#configuration)
-* [Startup](#startup)
-* [Post-installation sanity check](#post-installation-sanity-check)
-* [What's next](#whats-next)
+This document walks you through the installation of the Zonemaster *Backend* in the following sections:
 
-This document ends with these appendices:
+1. [Choices](#choices) lets you make all choices up front that this guide presumes you've made.
+2. [Prerequisites](#prerequisites) lets you check all assumptions made by this guide about pre-installed software.
+3. [Dependencies](#dependencies) shows you how to install all software that your Zonemaster *Backend* will depend on.
+4. [Installation](#installation) shows you how to install the *Backend* itself.
+5. [Configuration](#configuration) shows you how to configure the *Backend*.
+6. [Startup](#startup) shows you how to start the *Backend*.
+7. [Post-installation sanity check](#post-installation-sanity-check) lets you verify that your *Backend* has started correctly.
+
+This document ends with appendices:
 
 * [Quick installation using a Docker container](#quick-installation-using-a-docker-container)
 * [Administrative tasks](#administrative-tasks)
 
 
+## Choices

```

Comment:
Oh crap. Now I remember why I didn't do that before. It's because including numbers in the titles affects the section anchors. This way you can't add another section at the beginning or in between two other sections without messing with incoming links from other documents.

So which way hurts the least? Cemented section numbering or section titles without numbers?